tests/real/enterprise.rs

```diff
@@ -608,8 +608,8 @@ fn test_batch_of_jobs_can_be_initiated() {
     assert_eq!(s.failed, 0);
     // Docs do not mention it, but the golang client does:
     // https://github.com/contribsys/faktory/blob/main/client/batch.go#L17-L19
-    assert_eq!(s.success_callback_state, ""); // we did not even provide the 'success' callback
-    assert_eq!(s.complete_callback_state, ""); // the 'complete' callback is pending
+    assert_eq!(s.success_callback_state.to_string(), "Pending"); // we did not even provide the 'success' callback
```

Comment:
When you add `PartialEq` to this, you'll be able to just `assert_eq!` with `CallbackState::Pending` directly.